e2e/test/scenarios/dashboard-filters/dashboard-chained-filters.cy.spec.js

```diff
@@ -141,71 +144,106 @@ describe("scenarios > dashboard > chained filter", () => {
     });
   }
 
-  it.skip("should work for all field types (metabase#15170)", () => {
-    // Change Field Types for the following fields
-    cy.request("PUT", `/api/field/${PRODUCTS.ID}`, {
-      special_type: null,
-    });
-
-    cy.request("PUT", `/api/field/${PRODUCTS.EAN}`, {
-      special_type: "type/PK",
-    });
+  it(
+    "should work for all field types (metabase#15170)",
+    { tags: "@external" },
```

Comment:
It relies on the writable database since we have one that has a UUID field.https://github.com/metabase/metabase/blob/35653d30968c1fbe132666ed64194b3992e893ad/e2e/support/test_tables.js#L67.

If in the future, our sample database contains more fields, we could potentially use the sample database instead.